src/components/ProductCardList/index.tsx

```
@@ -1,26 +1,14 @@
-        const isInCart = shoppingCart.items.some(
-          (item) => item.product.id === product.id
-        );
-
-        return (
-          <ProductCard key={product.id} product={product} isInCart={isInCart} />
-        );
-      })}
+      {products &&
+        products.map((product) => (
+          <ProductCard key={product.id} product={product} />
+        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)